homeassistant/components/homekit_controller/climate.py

```diff
@@ -14,6 +14,7 @@
     HVAC_MODE_HEAT,
     HVAC_MODE_OFF,
     CURRENT_HVAC_OFF,
```

Comment:
`CURRENT_HVAC_OFF` isn't used any more so you will probably get told off by the commit bots for not removing it from the imports.

Reply:
all new territory for me, navigating github PR processes.   So it looks like you already removed CURRENT_HVAC_OFF.   Is there anything else I need to do?

Reply:
I haven't removed it so you'll need to

Reply:
I'm sorry @jc2k, but I am struggling to find my branch containing climate.py that I can edit.

Reply:
I found this version, but it is very different.  I guess from an HA repo copied quite a while ago? https://github.com/GaryOkie/home-assistant/blob/dev/homeassistant/components/homekit_controller/climate.py

Reply:
Duh, just found the secret menu to edit the dang file.   Hidden under the "..." > Edit.    Sheesh, sorry my noobness.